src/libraries/System.Diagnostics.TraceSource/src/System/Diagnostics/Switch.cs

```diff
@@ -50,11 +50,11 @@ private object InitializedLock
         /// <para>Initializes a new instance of the <see cref='System.Diagnostics.Switch'/>
         /// class.</para>
         /// </devdoc>
-        protected Switch(string displayName, string description) : this(displayName, description, "0")
+        protected Switch(string displayName, string? description) : this(displayName, description, "0")
         {
         }
 
-        protected Switch(string displayName, string description, string defaultSwitchValue)
+        protected Switch(string displayName, string? description, string defaultSwitchValue)
```

Comment:
`defaultSwitchValue` appears to allow `null` as well.

Reply:
Yes it is allow, but this value is used to init the `int SwitchSetting` which parses this value (indirectly) to int, where it could throw on null, so I thought it shouldn't be nullable

Reply:
You are right, it is really depend on implementation of how it is using the default value, so annotating its nullable for abstract Switch class but not null for derived classes